tests/integration/test_metrics.py

```diff
@@ -1148,3 +1148,51 @@ def test_span_metrics(
     for metric in span_metrics:
         assert metric["tags"]["span.description"] == expected_description
         assert metric["tags"]["span.group"] == expected_group
+
+
+def test_generic_metric_extraction(mini_sentry, relay):
+    PROJECT_ID = 42
+    mini_sentry.add_full_project_config(PROJECT_ID)
+
+    config = mini_sentry.project_configs[PROJECT_ID]["config"]
```

Comment:
nit: `add_full_project_config` returns the config
```suggestion
    config = mini_sentry.add_full_project_config(PROJECT_ID)["config"]
```

Reply:
Sorry, I did not see this comment in time.